### ISSUE TITLE
Missing artifacts on released file

### ISSUE THREAD

**Issue by marcgp** [cxpsemea/cxsast_custom_reporting#2]
**Describe the bug**
the released  zip file is missing mandatory artifacts.

**To Reproduce**
Please provide detailed steps to reproduce the error
1. Download released zip file
2. Extract it
4. see missing config folder

**Expected behavior**
released zip should have the following files
```
root
  - <executable file>
  - <config dir>
    - config.sample.ini
    - <templates dir>
        - ScanSummary.html
```

**Comment by marcgp** [comment]
changed release.sh in order to add missing artifacts. now release file should contain all proper information

**Comment by marcgp** [comment]
:information_source::information_source: **This issue has been fixed in [Release v1.0.0-next.2](https://github.com/cxpsemea/cxsast_custom_reporting/releases/tag/v1.0.0-next.2)**

**Comment by marcgp** [comment]
:information_source::information_source: **This issue has been fixed in [Release v1.0.0](https://github.com/cxpsemea/cxsast_custom_reporting/releases/tag/v1.0.0)**
